### PR TITLE
Enhancement/use new confirm controller

### PR DESCRIPTION
**Pull request by joefitter**

### CODE REVIEW

File: apps/complaints/fields.js [hunk 1]
```diff
@@ -135,13 +135,16 @@ module.exports = {
...
   },
   'complaint-date-month': {
-    validate: ['required', 'numeric']
+    validate: ['required', 'numeric'],
+    includeInSummary: false
   },
   'complaint-date-year': {
-    validate: ['required', 'numeric']
+    validate: ['required', 'numeric'],
+    includeInSummary: false
```
Comment (JoeChapman) [inline apps/complaints/fields.js]:
```js
dob: {
    label: 'Date of birth',
    hint: '',
    validate: ['required', /*numeric should be default*/],
    day: {
        label: 'Day'
    },
    month: {
        label: 'Mon',
        validate: 'alpha'
    },
    year: {
        label: 'Year',
        hint: 'dob.year.hint'
    }
}
```

File: apps/complaints/index.js [hunk 2]
```diff
@@ -211,6 +205,16 @@ module.exports = {
...
       }
     },
+    '/confirm': {
+      next: '/confirmation',
+      controller: controllers.confirm,
+      fieldsConfig: require('./fields'),
+      emailConfig: require('../../config').email,
+      customerEmailField: 'email-address',
+      locals: {
+        section: 'confirm'
```
Comment (JoeChapman) [inline apps/complaints/index.js]: btw, could a locals object contain literal mixins, i.e. `locals : {input: <input type...`?

Comment (JoeChapman) [inline apps/complaints/index.js]: I guess it's only actually useful if it's a compiled mixin.

Comment (joefitter) [inline apps/complaints/index.js]: yes, but they would been to be rendered with `{{{}}}`

**Comment by easternbloc** [comment]
Happy to merge but we are missing intro content on these emails. I'm also not happy with the email summary content.

It's quite clear this needs looking at by a content designer but lets put it on dev and we can use that to prompt change.

**Pull request merged**